src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -730,6 +730,26 @@ export var analyticsActionCauseList = {
     name: 'categoryFacetSearch',
     type: 'categoryFacet'
   },
+  /**
+   * The custom event that gets logged when an end-user expands a dynamic facet to see additional values.
```

Comment:
Why does the comment say "dynamic facet" while the type is `facet`
We decide to differentiate `facet` from `dynamicFacet` in the analytics (see dynamicFacetSelect for example)

Reply:
@fbeaudoincoveo should we document the metadata we send here?

Reply:
Ah, yes... we could say "expands a `DynamicFacet`" instead.

Concerning the event metadata, I realize a good, clean way to document this would be to expose the `IAnalyticsFacetMeta` interface in the public documentation, and append a sentence such as what follows to the `facetShowMore` / ` facetShowLess` description:

> When this event is logged, an object implementing the [`IAnalyticsFacetMeta`](@link IAnalyticsFacetMeta) interface is sent as custom data.

I'll create a Jira to refactor/improve public documentation in this entire file (`AnalyticsActionListMeta.ts`).

Reply:
@orobert-coveo for now we can leave this doc as it is; I'll do the required refactoring myself in https://coveord.atlassian.net/browse/JSUI-2501